action.yml

```diff
@@ -26,5 +26,5 @@ branding:
     icon: 'extension-icon.svg'
     color: 'blue'
 runs:
-  using: 'node16'
+  using: 'node20'
```

Comment:
Should this be a breaking change?

Reply:
actions will handle the node version right ? i mean what action uses should be no issue for the customers right, we will have nodemodules in the repo and the job will run this.

Reply:
we are asked to do too.